infobip_channels/core/http_client.py

```diff
@@ -18,6 +20,8 @@ def __init__(
         auth: Authentication,
         post_headers: RequestHeaders = None,
         get_headers: RequestHeaders = None,
+        put_headers: RequestHeaders = None,
+        delete_headers: RequestHeaders = None,
     ):
         """Create an instance of the _HttpClient class with the provided authentication
```

Comment:
Since we have more headers now, we should change this docstring a bit.
`Get and post headers can optionally be provided ...` should be changed to something like
```
Headers can optionally be provided, otherwise default instances will be created. These headers will be used as
defaults in the HTTP methods, unless new values are sent through method arguments.
```

Reply:
👍🏻